test/unit/karma.conf.js

```diff
@@ -102,7 +101,7 @@ module.exports = function (config) {
         exclude: [...alwaysExclude, ...isolatedTests],
         // test results reporter to use
         // available reporters: https://npmjs.org/browse/keyword/karma-reporter
-        reporters: ['mocha', 'coverage', 'json-result'],
+        reporters: ['mocha', 'coverage'],
```

Comment:
👍  I'm not sure that json-result gets used anywhere anymore. Maybe it was put in for some debugging?

Reply:
I know I put it in for debugging and I think I had something in the GHA involving it, but I don't think it gets used now. I certainly don't use it!